lib/src/filter/filter_options.dart

```diff
@@ -73,6 +82,7 @@ class FilterOptionGroup {
     result["createDate"] = createTimeCond.toMap();
     result["updateDate"] = updateTimeCond.toMap();
     result['containsEmptyAlbum'] = containsEmptyAlbum;
+    result['containsPathModified'] = containsPathModified;
```

Comment:
Lack of `this.containsPathModified = other.containsPathModified` in `merge` method.